packages/salesforcedx-utils-vscode/test/unit/commands/commandletExecutor.test.ts

```diff
@@ -6,37 +6,23 @@
  */
 
 import { expect } from 'chai';
-import * as proxyquire from 'proxyquire';
-import { assert, createSandbox, SinonSandbox, stub } from 'sinon';
-import { Progress } from 'vscode';
+import { assert, createSandbox, SinonSandbox, SinonStub, stub } from 'sinon';
+import { OutputChannel, Progress } from 'vscode';
+import * as vscode from 'vscode';
+import {
+  ChannelService,
+  LibraryCommandletExecutor,
+  NotificationService,
+  TelemetryService
+} from '../../../src';
 import { nls } from '../../../src/messages';
 import { ContinueResponse } from '../../../src/types';
-import { MockChannel, vscodeStub } from './mocks';
-
-const { ChannelService } = proxyquire.noCallThru()(
-  '../../../src/commands/index',
-  {
-    vscode: vscodeStub
-  }
-);
-
-const { LibraryCommandletExecutor } = proxyquire.noCallThru()(
-  '../../../src/index',
-  {
-    vscode: vscodeStub
-  }
-);
-
-const { NotificationService } = proxyquire.noCallThru()(
-  '../../../src/commands',
-  {
-    vscode: vscodeStub
-  }
-);
+import { MockChannel } from './mocks';
 
+const commandExecutionName = 'Test Command';
 class TestExecutor extends LibraryCommandletExecutor<{ success: boolean }> {
   constructor(outputChannel: MockChannel, private error?: Error) {
-    super('Test Command', 'test_command', outputChannel);
+    super(commandExecutionName, 'test_command', outputChannel as OutputChannel);
```

Comment:
Go ahead and leave this, but 'Test Command' didn't need to be refactored out.

Reply:
Yeah this shows my tendency to always put stings in a variable. Later on in the file the value was being used as part of an expect and I always want to use a variable vs copying string values.